data/Soumava-221B.json

```diff
@@ -0,0 +1,19 @@
+{
+"name": "Soumava Das",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am currently learning javascript",
+  "avatar": "https://github.com/Soumava-221B.png",
```

Comment:
Profile will be auto-imported from your github no need to add avatar link

Reply:
Okay, i will remove the avatar link and commit the changes